x-pack/plugins/security_solution/server/endpoint/services/actions/action_details_by_id.test.ts

```diff
@@ -97,7 +97,7 @@ describe('When using `getActionDetailsById()', () => {
                 action_id: '123',
                 data: {
                   command: 'isolate',
-                  comment: '5wb6pu6kh2xix5i',
+                  comment: expect.any(String),
```

Comment:
Yeah, I figure you had do this being that you switch to using that `faker` generator.

Reply:
I think I'll revert this faker change since it might break as you point out earlier. I just needed some long comments for testing truncates anyway.

Reply:
Maybe just use the `overrides` and pass in something like `'foo '.repeat(100)`